src/engine/test/health_test/engine-health-test/src/health_test/run_all.py

```diff
@@ -0,0 +1,84 @@
+from pathlib import Path
+from health_test.metadata_validate import run as metadata_validate_run
+from health_test.schema_validate import run as schema_validate_run
+from health_test.mandatory_mapping_validate import run as mandatory_mapping_validate_run
+from health_test.event_processing import run as event_processing_run
+from health_test.load_decoders import run as load_decoders_run
+from health_test.load_rules import run as load_rules_run
+from health_test.assets_validate import run as assets_validate_run
+from health_test.validate_successful_assets import run as validate_successful_assets_run
+from health_test.validate_non_modifiables_fields import run as validate_non_modifiables_fields_run
+from health_test.validate_custom_field_documentation import run as validate_custom_field_documentation_run
+from health_test.coverage_validate import run as coverage_validate_run
+from health_test.core import run as test_run
+from health_test.validate_custom_field_indexing import run as validate_custom_field_indexing_run
+from health_test.validate_event_indexing import run as validate_event_indexing_run
+from health_test.initial_state import run as init_run
```

Comment:
Change the format of import, something like:
```suggestion
from health_test import (
    metadata_validate,
    schema_validate,
    mandatory_mapping_validate,
    event_processing,
    load_decoders,
    load_rules,
    assets_validate,
    validate_successful_assets,
    validate_non_modifiables_fields,
    validate_custom_field_documentation,
    coverage_validate,
    core,
    validate_custom_field_indexing,
    validate_event_indexing,
    initial_state
)
```